package.json

```diff
@@ -1,12 +1,15 @@
 {
   "name": "gae-scaffold",
-  "version": "0.0.1",
+  "version": "0.0.2",
   "devDependencies": {
     "grunt": "~0.4.5",
     "grunt-appengine": "~0.1.5",
     "grunt-contrib-clean": "~0.5.0",
     "grunt-contrib-copy": "~0.5.0",
     "grunt-closure-soy": "~0.2.0",
-    "grunt-closure-tools": "~0.9.6"
+    "grunt-closure-tools": "~0.9.6",
+    "google-closure-library": "*",
+    "google-closure-compiler": "*",
+    "google-closure-templates": "*"
```

Comment:
SGTM. I initially didn't do this because of the slightly weird way [Closure](https://github.com/google/closure-library/releases) versioning is done, let me update the PR.